### DISCUSSION TITLE
Unit 3 Exercise 1 - Bank Note Authentication

### DISCUSSION THREAD

**Discussion by harishb00** [Lightning-AI/dl-fundamentals#95] [Exercises] [answered]
Hi @rasbt , 
I am working out the first exercise on bank note authentication in unit 3. The task is to find the `epochs` and `learning_rate` such that the training and validation accuracy reaches atleast 98%.

I achieve it. But what bothers me is that validation accuracy is greater than training accuracy. How's it even possible? Intuitively training accuracy should always be higher than the validation (unseen data) accuracy right? Could you please shed some light on this?

![image](https://github.com/Lightning-AI/dl-fundamentals/assets/43300971/3918eee5-90ca-4893-b0aa-8af1e01e0d61)

I used `epochs=30` and `lr=0.53`

**Comment by rasbt** [comment, marked as answer]
That's an interesting one. It's rare but it can happen. This could be due to randomness (Ultimately, it's still a relatively small dataset, and there are only ~270 instances in the validation set). E.g., assume the model does not overfit and performs equal on the training and validation set. You will always have some slight variation based on which exact examples are in the training and the validation (i.e., how you split the dataset by varying the random seed, for example).

Another explanation is that you have tuned the model so much (via hyperparameter choices) to improve the validation accuracy that it is slightly overtuned on the validation set. This is why it's important to have an additional test set for the final evaluation (If I recall correctly, I covered it later in the course.)

**Reply by harishb00** [reply]
Nice explanation. Both of the points make sense. 
Thanks a lot @rasbt for answering my questions.
❤️ from India.
